automation/run-tests.sh

```diff
@@ -68,17 +71,31 @@ function install_nmstate {
 }
 
 function run_tests {
-    docker_exec "
-      cd /workspace/nmstate &&
-      pytest \
-        --verbose --verbose \
-        --log-level=DEBUG \
-        --durations=5 \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/libnmstate \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/nmstatectl \
-        --cov-report=html:htmlcov-integ \
-        tests/integration \
-    ${nmstate_pytest_extra_args}"
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "tox_code_style" ];then
```

Comment:
I just realized that for all, it will not execute all tests when an earlier one fails. IMHO it would be nice to run the integration tests first and ideally ensure that all tests run even with the earlier fail. If we can run the integration tests via tox, we could use just one tox call to run all tests, i.e. 'tox -e integration,flake8,pylint,check-py27, ...' to avoid early exits.

Reply:
Seems like this belongs in #233